lib/Form/Type/FieldType/TimeFieldType.php

```diff
@@ -44,21 +45,21 @@ public function getParent()
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->addModelTransformer(new FieldValueTransformer($this->fieldTypeService->getFieldType('eztime')));
+        $builder
+            ->addViewTransformer(new FormattedTimeToTimestampTransformer($options['with_seconds']))
+            ->addModelTransformer(new TimeValueTransformer());
     }
 
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
-        if ($options['with_seconds']) {
-            $view->vars['attr']['step'] = 1;
-        }
+        $view->vars['attr']['data-seconds'] = (int) $options['with_seconds'];
```

Comment:
Ok, so is this option a `bool` (inferring from `FormattedTimeToTimestampTransformer` constructor call) or an `int`?
The "thing" on the other side, using view variables should be able to handle bool.
But I don't know if it's possible for forms, if not then this is fine.

Reply:
It a `bool` value but we need to present is as either `0` or `1` as `data-*` HTML attributes do not support boolean values.